packages/eddsa-proof/package.json

```diff
@@ -54,7 +48,7 @@
         "@ethersproject/bytes": "^5.7.0",
         "@ethersproject/keccak256": "^5.7.0",
         "@zk-kit/eddsa-poseidon": "0.11.0",
-        "@zk-kit/utils": "1.0.0-beta.1",
+        "@zk-kit/utils": "workspace:^",
```

Comment:
This won't work when publishing the package. It should be `1.0.0-beta.2`

Reply:
I was wondering how you handle these cases?
If the ci tests depends on a local package that hasn't been published yet... do you release it manually under beta yourself first?

Reply:
Even if you specify its version, it always uses the local version locally. So the CI tests should use local bundles generated with `yarn build:libraries`.